packages/web-lib/utils/isZero.test.ts

```diff
@@ -0,0 +1,33 @@
+import {isZero} from '@yearn-finance/web-lib/utils/isZero';
+
+describe('isZero function', (): void => {
+	test('returns false when no argument given', (): void => {
+		expect(isZero()).toBe(false);
+	});
+
+	test.each([
+		[0, true],
+		['0', true],
+		['0.0', true],
```

Comment:
I don't think it's an overkill, it's a great one! The idea here is that this is the helper function that you can give anything and it will let you know if it's zero :)

Added support for 0,0 with https://github.com/yearn/web-lib/pull/261/commits/49697199ff0416c74333701a6c86d5eefeaf2d46